src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -467,14 +470,70 @@ void getIndividualPreferredHearingChannel() {
         assertNull(result);
     }
 
-    @DisplayName("getIndividualInterpreterLanguage Test")
-    @Test
-    void getIndividualInterpreterLanguage() {
-        // TODO Finish Test when method done
-        HearingOptions hearingOptions = HearingOptions.builder().build();
+    @DisplayName("When language passed in should return correct LOV format")
+    @ParameterizedTest
+    @CsvSource({"Acholi,ach", "Afrikaans,afr", "Akan,aka", "Albanian,alb", "Zaza,zza", "Zulu,zul"})
+    void getIndividualInterpreterLanguageTest(String lang, String expected) {
+        HearingOptions hearingOptions = HearingOptions.builder()
+            .languageInterpreter("Yes")
+            .languages(lang)
+            .build();
         String result = HearingsPartiesMapping.getIndividualInterpreterLanguage(hearingOptions);
+        assertThat(result).isEqualTo(expected);
+    }
 
-        assertNull(result);
+    @DisplayName("When sign language passed in should return correct LOV format")
+    @ParameterizedTest
+    @CsvSource({"American Sign Language (ASL),americanSignLanguage", "Hands on signing,handsOnSigning", "Deaf Relay,deafRelay", "Palantypist / Speech to text,palantypist"})
+    void getIndividualInterpreterSignLanguageTest(String signLang, String expected) {
+        List<String> arrangements = Collections.singletonList("signLanguageInterpreter");
+        HearingOptions hearingOptions = HearingOptions.builder()
+            .arrangements(arrangements)
+            .signLanguageType(signLang)
+            .build();
+        hearingOptions.wantsSignLanguageInterpreter();
+        String result = HearingsPartiesMapping.getIndividualInterpreterLanguage(hearingOptions);
+        assertThat(result).isEqualTo(expected);
+    }
+
+    @DisplayName("When hearing type paper then return LOV not attending")
+    @Test
+    void getIndividualPreferredHearingChannelPaperTest() {
```

Comment:
can you make a test for when both hearingType is null and hearingSubtype is null